jl4/examples/ok/tests/float.nlg.golden

```diff
@@ -0,0 +1,26 @@
+`pi`
+`euler`
+`pi`
+the sum of `pi` and `euler`
+the difference between `pi` and the product of `pi` and `euler`
+the result of dividing `pi` by `euler`
+the result of dividing 5 by 0
+the result of dividing 5 by 0
+the result of 10 modulo 5
+the result of 10 modulo 5
+the result of 0.5 modulo 5
+the result of 5 modulo 0
+5 is equal to 5
+5 is equal to 5
+the sum of 0.1 and 0.2
+the sum of 0.1 and 0.2 is equal to 0.3
+the sum of `ROUND` with `pi` and `FLOOR` with `pi`
+the sum of `pi` and the product of `ROUND` with `pi` and `pi`
+the product of `FLOOR` with `pi` and `FLOOR` with `pi`
+`CEILING` with `pi`
+`IS_INTEGER`
+`IS_INTEGER` with `pi`
+`IS_INTEGER` with 2
+`IS_INTEGER` with 2
+`IS_INTEGER` with 2.00000001
+consider the case distinctions of `IS_INTEGER` with `pi` :  when `TRUE` has  then `pi`. when `FALSE` has  then the sum of 0.1 and 0.2
```

Comment:
i guess this would be an oversight for nlg. it seems as though this always generates a `has` for constructors, even though `TRUE` doesn't have any fields.